client/config/config.go

```diff
@@ -36,6 +36,36 @@ type ClientConfig struct {
 	Trace          bool   `mapstructure:"trace" json:"trace"`
 }
 
+// TODO Validate values in setters
+func (c *ClientConfig) SetChainID(chainID string) {
+	c.ChainID = chainID
+}
+
+func (c *ClientConfig) SetKeyringBackend(keyringBackend string) {
+	c.KeyringBackend = keyringBackend
+}
+
+func (c *ClientConfig) SetOutput(output string) {
+	c.Output = output
+}
+
+func (c *ClientConfig) SetNode(node string) {
+	c.Node = node
+}
+
+func (c *ClientConfig) SetBroadcastMode(broadcastMode string) {
+	c.BroadcastMode = broadcastMode
+}
+
+func (c *ClientConfig) SetTrace(trace string) error {
```

Comment:
@amaurym  Essentially, SetTrace can return error if it is unable to convert string to bool. I can implement similar approach in SetKeyringBackend to check if keyringBackend can hold one of  5-6 values, right?